test/dummy_webpacker3/config/webpack/environment.js

```diff
@@ -1,3 +1,5 @@
 const { environment } = require('@rails/webpacker')
```

Comment:
why support webpacker 3? it's too old!

Reply:
Because decisions on what to support are above our pay-grade. :joy: